cpp/include/raft/core/cublas_macros.hpp

```diff
@@ -108,6 +122,9 @@ inline const char* cublas_error_to_string(cublasStatus_t err)
     }                                                                \
   } while (0)
 
+/**
+ * @}
+ */
```

Comment:
Unrelated but was wondering if we have a gh issue for the rename mentioned just below?